theme/ItaliaTheme/Addons/customerSatisfaction.scss

```
@@ -37,4 +37,20 @@
       max-height: 800px;
     }
   }
+  .comment {
+    @media (max-width: #{map-get($grid-breakpoints, lg)}) {
```

```suggestion
    @media (max-width: #{map-get($grid-breakpoints, lg) - 1px}) {
```

Solitamente quando si usa `max-width` è meglio mettere 1px in meno rispetto al breakpoint per non creare sovrapposizioni di stili se lo schermo è largo esattamente come il breakpoint.